apps/wing/src/commands/test/test.test.ts

```diff
@@ -358,6 +358,39 @@ describe("retry option", () => {
     const retryLogs = logSpy.mock.calls.filter((args) => args[0].includes("Retrying"));
     expect(retryLogs.length).toBe(3);
   });
+
+  test("wing test --parallel [batch]", async () => {
+    const outDir = await fsPromises.mkdtemp(join(tmpdir(), "-wing-batch-test"));
+
+    process.chdir(outDir);
+
+    fs.writeFileSync(
+      "t1.test.w",
+      `
+      bring cloud;
+      test "t1" {
+        assert(true);
+      }
+    `
+    );
+    fs.writeFileSync(
+      "r2.test.w",
+      `
+      bring cloud;
+      test "t2" {
+        assert(true);
+      }
+    `
+    );
+
+    await wingTest(["t1.test.w", "t2.test.w"], {
+      clean: true,
+      platform: [BuiltinPlatform.SIM],
+      parallel: 1,
+    });
+    const batchLogs = logSpy.mock.calls.filter((args) => args[0].includes("Processing batch"));
+    expect(batchLogs.length).toBe(0);
```

Comment:
The "Processing batch" message is not ever printed so this test doesn't really verify anything

Reply:
yes, I fixed it just now, I had to go somewhere last evening, and I got to it only now